src/subscription/subscribe.js

```diff
@@ -220,16 +249,23 @@ export function createSourceEventStream(
     info
   );
 
-  if (subscription instanceof Error) {
-    throw subscription;
-  }
+  return Promise.resolve(subscription).then(resolvedSubscription => {
```

Comment:
`buildExecutionContext` above may throw a GraphQLError if client-provided values don't coerce properly - those should be properly formatted in the response via returning a rejected promise. You can test this by passing bad variable values along with a query.

The fix should probably be to wrap `return new Promise(resolve => {})`starting above line 196, ending in `resolve(subscription)` and followed by this `.then()`